### DISCUSSION TITLE
Confused about naming / versioning

### DISCUSSION THREAD

**Discussion by definiteIymaybe** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#329] [Q&A] [answered]
So I see a new release tagged 1.04 today. I check my installed version. It says it's AlDente 2.2.

`brew info aldente` is also offering me Al Dente 2.2. I go [releases](https://github.com/davidwernhart/AlDente/releases) and I see something called Al Dente classic. Wasn't able to find anything on differences. The "Download Free" button on the [website](https://apphousekitchen.com/pricing/) gives Al Dente 2.2. That's how I know version 2 is Classic.

The README.md only makes distinction between Pro and Free, no comments on the Classic / non-Classic versions. Is it Free = Classic and Pro = non-Classic?

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @definiteIymaybe,

the README got updated and explains the reason behind the naming of AlDente Classic:

"AlDente Classic is our original version of AlDente. First introduced in 2020 and further developed until March 2021. With a complete rewrite of the code of AlDente and the introduction of AlDente Pro, we decided to rename the original AlDente in AlDente Classic and use the name AlDente (Free) or AlDente Pro with synchronous versioning between these two from now on."

In short: AlDente 1.04 is the newest version of the free version of AlDente with numerous bug fixes and will be updated from now on simultaneously with AlDente Pro. AlDente Classic is the original AlDente and will not receive updates from now on.

The website got updated as well.

**Comment by dmelgar** [comment]
Looks like AlDente GitHub repo does not represent the source code in any recent release. The source code here is stagnant since the 2.0 Alpha release and does not contain the changes described to use SMC key CH0C for Apple Silicon. 
Releases after 2.0 alpha, including 2.1, 2.2 and 1.0.4 are NOT represented in this repo.

Each release includes "assets" which presumably represent the source code to build that binary. That is untrue and not the case. The source code linked as an asset for all releases is the 2.0 alpha release and has not changing between releases.

This appears intentionally misleading. This project has not been open source since 2.0 alpha if not earlier.
Github is being used as a way to advertise a closed source app since the AlDente Classic 2.0 release without stating its closed source.

**Reply by MatthiasKerbl** [reply]
Hi @dmelgar,

We are sorry if we broke some kind of Github rule by publishing our new versions of AlDente as a closed source app. It was never our intention to mislead or deceive anyone by using the newest versions of AlDente. Thank you for pointing out this mistake of ours. I have updated our readme to make it clearer for everyone when the change from open source to closed source happened.

If you have any other suggestions on improving our communication and transparency, let us know.

Kind regards,
Matthias
